cmd/binapi-generator/main.go

```diff
@@ -18,53 +18,64 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
-	"strings"
-	"unicode"
 
 	"github.com/sirupsen/logrus"
 
 	"go.fd.io/govpp/binapigen"
 	"go.fd.io/govpp/binapigen/vppapi"
 	"go.fd.io/govpp/version"
+	_ "go.fd.io/govpp/binapigen/plugins"
 )
 
 func init() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "USAGE\n")
-		fmt.Fprintf(os.Stderr, "  Parse API_FILES and generate Go bindings\n")
-		fmt.Fprintf(os.Stderr, "  Provide API_FILES by file name, or with full path including extension.\n")
-		fmt.Fprintf(os.Stderr, "  %s [OPTION] API_FILES\n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "GoVPP api bindings generation CLI\n")
```

Comment:
I think the term "GoVPP api bindings" is not correct here. The tool is generating Go bindings for VPP API.